src/main/java/gg/uhc/uhc/modules/team/requests/RequestManager.java

```diff
@@ -138,6 +138,7 @@ public boolean finalizeRequest(int id, AcceptState accepted) {
 
             for (OfflinePlayer p : players) {
                 team.addPlayer(p);
+                p.setWhitelisted(true);
```

Comment:
I don't like the idea of a side effect of whitelisting a team request member, especially if it isn't documented. Can this be documented and added as either 1) a config option or 2) a flag on accepting the request to whitelist/not whitelist depending on what makes more sense logically